code/__HELPERS/unsorted.dm

```diff
@@ -449,7 +449,7 @@ Turf and target are separate in case you want to teleport some distance from a t
 	Gets all contents of contents and returns them all in a list.
 */
 
-/atom/proc/GetAllContents(var/T)
+/atom/proc/GetAllContents(var/T, ignore_flag)
```

Comment:
call it ignore_flag_1 for now, since someone may need a new flags var later, and it matches our scheme of naming the flag vars.